src/OpenLoco/OpenLoco.cpp

```diff
@@ -951,6 +952,7 @@ namespace OpenLoco
         catch (const std::exception& e)
         {
             std::fprintf(stderr, "Unable to autosave game: %s\n", e.what());
+            Ui::showMessageBox("Unable to Autosave:", e.what());
```

Comment:
Is this actually worthy of a showMessasgeBox. Autosaves only happen when you are already in game and everything is loaded. Therefore the normal interface can be used to show the error.

Reply:
removed this